src/SLCore/Core/IRpcDebugger.cs

```diff
@@ -36,31 +36,29 @@ public interface IRpcDebugger
 internal class RpcDebugger : IRpcDebugger
 {
     private readonly IFileSystem fileSystem;
-    public string LogFilePath { get; set; } = null;
+    private readonly string logFilePath;
 
     [ImportingConstructor]
-    public RpcDebugger() : this(new FileSystem())
+    public RpcDebugger() : this(new FileSystem(), DateTime.Now)
     {
     }
 
-    internal /* for testing */ RpcDebugger(IFileSystem fileSystem)
+    internal /* for testing */ RpcDebugger(IFileSystem fileSystem, DateTime dateTime)
     {
         this.fileSystem = fileSystem;
+
+        logFilePath = $"{dateTime.ToString("yyyy-MM-dd_HHmmssffff")}.txt";
```

Comment:
what is the base directory for this? is it going to be where the vsix is installed? It would be better if it was a constant folder (appdata, temp, etc.)